apps/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 # along with this library; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-if(NOT ${libwebsockets_VERSION} VERSION_LESS 2.0.0)
+if(NOT ${Libwebsockets_VERSION} VERSION_LESS 2.0.0)
```

Comment:
shouldn't rockets/jsonrpc/http.* not also be excluded from library if version is too old?